python/lsst/ts/ess/mock/ess_temperature_4ch_c.py

```diff
@@ -0,0 +1,34 @@
+# This file is part of ts_Dome.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["EssTemperature4ChC"]
+
+
+class EssTemperature4ChC:
+    """Mock class that contains the 4 channel temperature data.
+    """
+
+    def __init__(self):
+        self.timestamp = None
+        self.TemperatureC01 = None
```

Comment:
Consider changing that to array. Also please use lover case start for variables (temperatureC01) as upper cases are reserved for classes.

Reply:
I agree that that would be better but I don't know what the internal structure of this class is so until I know that I cannot change this code. I am merely following what's done in the ESS_Temperature4Ch_Publisher.sh file. And I will work with our external developer to improve the code in this respect, including camel casing of the variable names.